src/main.py

```diff
@@ -118,15 +131,66 @@ def test(am, test, f, output):
         print(f'3.2. Loading aspect model from {output}f{f}.model for testing ...')
         am.load(f'{output}/f{f}.')
         print('3.3. Testing aspect model ...')
-        pairs = am.infer_batch(reviews_test=test, h_ratio=params.settings['test']['h_ratio'], doctype=params.settings['prep']['doctype'])
+        pairs = am.infer_batch(reviews_test=test, h_ratio=params.settings['test']['h_ratio'], doctype=params.settings['prep']['doctype'], output=output)
         pd.to_pickle(pairs, f'{output}f{f}.model.pred.{params.settings["test"]["h_ratio"]}')
 
-def evaluate(input, output):
+
+# TODO: Refactor this to convert Bert pairs to LADy pairs
+def evaluate_bert(input: str, output: str):
```

Comment:
@3ripleM we should not have this here. the main should not be customized for a specific model

Reply:
Thats correct and I totally agree with you. this is a monkey patch for the purpose of the paper then ma and farniam or you can make it cleaner (we should just write a function to convert bert pairs to our pairs)

Reply:
put it sw inside the bert class then